src/integrators/wrapper_integrator.cpp

```diff
@@ -0,0 +1,80 @@
+// Copyright (c) 2019-2020, Lawrence Livermore National Security, LLC and
+// other Serac Project Developers. See the top-level LICENSE file for
+// details.
+//
+// SPDX-License-Identifier: (BSD-3-Clause)
+
+# include "wrapper_integrator.hpp"
+
+LinearToNonlinearFormIntegrator::LinearToNonlinearFormIntegrator( mfem::LinearFormIntegrator & f,
+    mfem::ParFiniteElementSpace * trial_fes)
+  :
+  m_f(f), m_trial_fes(trial_fes)
```

Comment:
Run make style on this PR please. It should work on any checked in host-config now

Reply:
What do you mean by checked in host-config? So just to clarify, the style changes have been made recently?

Reply:
In the repo there is a directory with host-configs that have shared libraries you can use on the network.  For example, host-configs/rzgenie-toss_3_x86_64_ib-gcc@8.1.0.cmake points at our public libraries with all the dev tools enabled.

./config-build.py -hc host-configs/rzgenie-toss_3_x86_64_ib-gcc@8.1.0.cmake
cd build-rzgenie-toss_3_x86_64_ib-gcc@8.1.0-debug
make style